harness/setup.py

```diff
@@ -43,8 +43,7 @@
         "python-dateutil",
         "pytz",
         "tabulate>=0.8.3",
-        # det preview-search "pretty-dumps" a sub-yaml with an API added in 0.15.29
```

Comment:
versions older than 0.15.78 don't install with python 3.8, so this pin is no longer relevant.